index.d.ts

```diff
@@ -1683,7 +1690,6 @@ declare namespace Eris {
     position: number;
     type: Exclude<ChannelTypes, 1 | 3>;
     constructor(data: BaseData, guild: Guild);
-    createInvite(options?: CreateInviteOptions, reason?: string): Promise<Invite & InviteWithoutMetadata<null>>;
```

Comment:
This needs to be added back

Reply:
GuildChannel doesn't have createinvite, only textchannel does 🤔

Reply:
Or is something i'm missing

Reply:
Yup that's my bad